### DISCUSSION TITLE
unique columns [Jul 21]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#2872] [Discord Threads]
> 🙋‍♂️ **lublak**

**Comment by o1lab** [comment]
> 🙋‍♂️ **lublak**  
#### currently i can create unique values with sql. Also over multiple columns.  
#  
> 👨‍🦲 **dstala**  
#### do you wish to concatenate column values? or create something like sql index?  
#  
> 🙋‍♂️ **lublak**  
#### That the value only occurs once or, in the case of several columns, that the combination of values only occurs once.  
#  
> 🙋‍♂️ **lublak**  
#### <@857998597474484295> is this another feature request? Or is it already possible?  
#  
> 💁 **o1lab**  
#### creating unique is not possible at the moment - however we would like to do this.. please raise a feature request  
#  
> 💁 **o1lab**  
#### which database are you using ?  
#  
> 🙋‍♂️ **lublak**  
#### I mostly use PostgreSQL  
#  
> 🙋‍♂️ **lublak**  
#### I create I feature request tomorrow (on GitHub)  
#  
> 💁 **o1lab**  
#### tahnks  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/999309860913021078)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
